common-theme/assets/scripts/time-total.js

```diff
@@ -0,0 +1,37 @@
+class TimeTotal extends HTMLElement {
+  constructor() {
+    super();
+    this.attachShadow({ mode: "open" });
+  }
+
+  connectedCallback() {
+    this.showTotalTime();
+  }
+
+  getTotalMinutes = (timeElements) =>
```

Comment:
This doesn't include videos and video playlists, some of which are multiple hours... I guess there's probably not a cute way for us to do so, but this feels like a problem...

Reply:
you can add it on the block manually in the prep view, which is what I have done - I think it might be nice to winkle it out of the link but not really vital